example/crs/mellanox.com_v1alpha1_nicclusterpolicy_cr-full.yaml

```diff
@@ -0,0 +1,118 @@
+# Copyright 2020 NVIDIA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ##### Note #####
+# This example contains all the components supported as a reference.
+# User should edit the example and keep only the required components.
+#
```

Comment:
```suggestion

```

nit: do we need this example text or should we rather remove it?

Reply:
This example is a reference, that was asked from QA team to easily get the components versions, taking into account that in the future these versions will not be available in the Helm values.
This configuration probably is not a real life scenario.